src/transforms/babel.js

```diff
@@ -203,21 +245,31 @@ async function getBabelRc(asset, isSource) {
 
     // If specified as an array, override the config with the one specified
     if (Array.isArray(babelify) && babelify[1]) {
-      return babelify[1];
+      return {
+        version: 6,
```

Comment:
I guess we should test this the same way as above - it's possible that there could be babel 7 plugins in here too.

Reply:
Alright, just thought legacy, people won't use that for babel 7. But sure I'll make it check the plugins